Sources/Core/TextureTable.cpp

```diff
@@ -0,0 +1,18 @@
+#include "TextureTable.hpp"
+
+using namespace LWGC;
+
+TextureTable::TextureTable(void)
```

Comment:
Remove these useless constructor and destructor and use [c++ default ones](https://en.cppreference.com/w/cpp/language/default_constructor)